build/spas.ts

```diff
@@ -1,6 +1,7 @@
 import fs from "fs";
 import path from "path";
 import frontmatter from "front-matter";
+import klawSync from "klaw-sync";
```

Comment:
Sorry I thought I got rid of `klawSyn`c everywhere. We use `fdir` in most places. Would you mind changing it (and potentially replace `klawSync` in `tool/cli.ts`.

Reply:
Sure

Reply:
Fixed in fc93362cd.